config.example.toml

```diff
@@ -59,6 +59,17 @@ hyperswitch = { master_key = "feffe9928665731c6d6a8f9467308308feffe9928665731c6d
 # token = "hvs.abc" # The secret token to access and communicate with the vault
 
 # TLS server within axum
-# [tls]
-# certificate = "cert.pem" # path to the certificate file (`pem` format)
-# private_key = "key.pem" # path to the private key file (`pem` format)
+[tls]
+certificate = "cert.pem" # path to the certificate file (`pem` format)
+private_key = "key.pem" # path to the private key file (`pem` format)
+
+# Api client
+[api_client]
+client_idle_timeout = 90 # timeout for idle sockets being kept-alive
+pool_max_idle_per_host = 10 # maximum idle connection per host allowed in the pool.
+identity = "" # identity to be used for client certificate authentication in mtls.
+
+# Configuration for the Key Manager Service
+[key_manager]
+url = "http://localhost:5000" # URL of the encryption service
+cert = "" # Represents a server X509 certificate for mtls
```

Comment:
Should we take the certificate as the input? or a file path instead?

Reply:
which one is convenient and secure in deployment environments? if we pass direct certificate, then we'll have to kms encrypt it. But if its file path then the file has to be kept secure. what do u suggest?